extensions/notebook/src/dialog/configurePython/pickPackagesPage.ts

```diff
@@ -158,9 +158,9 @@ export class PickPackagesPage extends BasePage {
 				this.requiredPackagesTable.data = requiredPkgVersions.map(pkg => [pkg.name, pkg.existingVersion ?? '-', pkg.requiredVersion]);
 				this.model.packagesToInstall = requiredPackages;
 			} else {
-				this.instance.showErrorMessage(localize('msgUnsupportedKernel', "Could not retrieve packages for kernel {0}", kernelName));
+				this.instance.showInfoMessage(localize('msgUnsupportedKernel', "No packages are required by default for the kernel '{0}'", kernelName));
```

Comment:
The original message seemed to imply that there was some issue fetching packages...is that not the case anymore? I suppose it probably would have thrown if that was the case...

If so then I'd suggest updating the loc message ID too

Reply:
We report issues with fetching packages elsewhere. This check is just a shortcut for installing dependencies for known kernels. If we don't have any required packages for the specified kernel, then reporting it as an error is confusing for the user.